### ISSUE TITLE
[Bug] Build issues with Angular project (webpack)

### ISSUE THREAD

**Issue by luchillo17** [rguerreiro/device#26]
Webpack in an angular project seems to have issues with json imports when the extension of the file is not added in the require, as well as the package `useragent` having issues with `yamlparser` being in the wrong dependency property (is in dev deps, not in normal ones where it should).

![screenshot from 2018-02-15 17-50-31](https://user-images.githubusercontent.com/9698639/36285347-2e6e9c06-1279-11e8-8439-b2b37155c940.png)

**Comment by rguerreiro** [comment]
I'll have to check it out. This package was made many years ago, in a time we did not used webpack. Which version of webpack are you using?

**Comment by luchillo17** [comment]
Was using angular 5 which means something like webpack 4, but the issue i tracked down to one of your deps, i made a PR for it to a version of such dep which i made another PR for, but they didn't approved it.
I'm not using this lib any more, if you want we can close this issue.

**Comment by davidfurlong** [comment]
got same issue.
```
ERROR in ./node_modules/useragent/lib/update.js
Module not found: Error: Can't resolve 'yamlparser' in '/Users/DVF/Projects/deedmob/client/node_modules/useragent/lib'
 @ ./node_modules/useragent/lib/update.js 15:11-32
 @ ./node_modules/useragent/index.js
 @ ./node_modules/device/lib/device.js
 @ ./node_modules/device/index.js
 @ ./src/helpers/getContextInfo.ts
```

**Comment by davidfurlong** [comment]
Basically the way this library is used requires a `npm install yamlparser` -- see https://www.npmjs.com/package/useragent

**Comment by rguerreiro** [comment]
It was fixed on #33
